homeworks/oleksandr.khodakivskyi_Moo0oony/GamesProject/styles/main.css

```diff
@@ -59,10 +58,17 @@ main {
   line-height: 21px;
 }
 
+.game_image img {
+  width: 90px;
+  height: 90px;
+  border-radius: 8px;
+  object-fit: cover;
+}
+
 .game_info {
   display: flex;
   flex-direction: column;
-  margin: 6.5px 0 6.5px 12px;
+  margin: 0 0 6.5px 12px;
```

Comment:
Minor change could improve visual representation of your cards 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/d05abb4c-c0fe-477e-bc42-da55d633cdf9)